onadata/apps/messaging/README.md

```diff
@@ -0,0 +1,15 @@
+Messaging
+=========
+
+Adds support to create and view messages via an API.
```

Comment:
This needs more details, what types of messages, where are they getting sent, why would anyone want to use this?